pennylane/operation.py

```diff
@@ -799,6 +799,68 @@ def eigvals(self):
     def _matrix(cls, *params):
         return np.diag(cls._eigvals(*params))
 
+class Channel(Operation):
+    r"""Base class for quantum channels supported by a device.
+
+    As with :class:`~.Operation`, the following class attributes must be
+    defined for all operations:
```

Comment:
```suggestion
    As with unitary gates, the following class attributes must be
    defined for all channels:
```

Reply:
(fly-by-comment: I would leave this as a class link, since it is specifically referring to the parents class attributes)